Week1/Homework/select-star.js

```diff
@@ -0,0 +1,37 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect();
+
+var select_queries = [
+  'select Country.name from Country where Country.population >= 8000000 order by name ASC',
```

Comment:
You don't need to have `Country.` thing as long as everything is in the same table